pkgs/development/libraries/kde-frameworks/kde-wrapper.nix

```diff
@@ -61,13 +61,9 @@ stdenv.mkDerivation {
         fi
     done
 
-    ln -s "$env/share" "$out"
```

Comment:
If I understand correctly exactly this would happen:

1. `buildEnv` №1 (that is, `kdeWrapper`) makes `/share` including all packages propagated with `propagatedUserEnvPkgs`;
2. `buildEnv` №2 (`environment.systemPackages`) has №1 as its input (including symlinks to some propagated `foo`) and also `foo` its second input. Here we get a clash!